services/maintenance/README.md

```diff
@@ -0,0 +1,29 @@
+# portainer stack
```

Comment:
The portainer stack under maintenance??

Reply:
well, I did not touch this part, remember we did this together when copying the maintenance stack out of simcore. Shall I take care of it?